test/constraint.jl

```diff
@@ -563,6 +563,13 @@ end
             (Vector{AffExpr}, MOI.PositiveSemidefiniteConeSquare),
             (Vector{QuadExpr}, MOI.RotatedSecondOrderCone)])
     end
+    @testset "get and set dual start" begin
+	model = Model()
+	@variable(model, x)
+        con = @constraint(model, 2x <= 1)
+	set_dual_start_value(con, 2)
+	@test dual_start_value(con) == 2.0
```

Comment:
Does `@test 2.0 == @inferred dual_start_value(con)` pass ?

Reply:
no the test is failing.
Should it return 0 instead of `Nothing` when no value is set. At least will be type stable then